### PR TITLE
Make Googlnet & InceptionNet scriptable

### PR DESCRIPTION
**Pull request by eellison**
Based on the conclusion from https://github.com/pytorch/vision/issues/1273: we are always returning a tuple when scripting. 

This was blocked by https://github.com/pytorch/pytorch/pull/26683.

### CODE REVIEW

**Comment by eellison** [comment]
blocked by https://github.com/pytorch/pytorch/issues/26437

File: test/test_models.py [hunk 1]
```diff
@@ -55,7 +53,7 @@ def check_script(self, model, name):
             tb = traceback.format_exc()
             scriptable = False
             msg = str(e) + str(tb)
-        self.assertEqual(torchub_models[name], scriptable, msg)
+        self.assertEqual(scriptable, scriptable, msg)
```
Comment (fmassa) [inline test/test_models.py]: I'm not sure this change is what you want to do?

Comment (eellison) [reply test/test_models.py]: Since they are all scriptable now, we don't need to access `torchub_models` and just assert that `scriptable` is True

Comment (fmassa) [reply test/test_models.py]: Can you then make
```python
self.assertEqual(scriptable, True, msg)
```

Comment (eellison) [reply test/test_models.py]: oh oops - good catch

**Comment by fmassa** [comment]
Test failures look legit

**Pull request merged**